benchmarks/benchmarks/tracking.py

```diff
@@ -28,8 +28,3 @@ def time_singleaxis(self):
                                 max_angle=60,
                                 backtrack=True,
                                 gcr=0.45)
-
-    def time_tracker_singleaxis(self):
-        with np.errstate(invalid='ignore'):
-            self.tracker.singleaxis(self.solar_position.apparent_zenith,
-                                    self.solar_position.azimuth)
```

Comment:
In addition to this benchmark function, the definition of `self.tracker` on line 20 should also be removed as that's what is actually emitting the deprecation warning, see https://github.com/pvlib/pvlib-python/runs/7828546496?check_suite_focus=true#step:5:189

Reply:
thank you for clarifying, i'll make the changes